ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -443,7 +486,7 @@ void OwInterface::initialize()
 
   if (not initialized) {
 
-    for (auto name : LanderOpNames) {
+    for (const string& name : LanderOpNames) {
```

Comment:
Curious why...

Reply:
I am by no means of the _`auto` all the things_ mindset, but in this case it's simply unnecessary to restate what data type that `LanderOpNames` contains when the compiler can infer it for you. The more practical reason to use `auto` instead of `string` here is to future-proof the code. If the contained datatype of `LanderOpNames` were to be changed by a future PR, this line won't need to be modified as a result.